src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -196,6 +203,16 @@ public static ConfigDef newConfigDef() {
           Width.SHORT,
           LOGS_DIR_DISPLAY
       );
+
+      // define avro codec
+      configDef.define(AVRO_CODEC_CONFIG,
+          Type.STRING,
+          AVRO_CODEC_DEFAULT,
+          Importance.LOW,
+          AVRO_CODEC_DOC,
+          group, ++orderInGroup, Width.LONG,
```

Comment:
In the current code style (outside AK) we use one argument per line in multi-line method calls. A bit wasteful in terms of space but easier to maintain and not miss an argument.